requirements.txt

```diff
@@ -1,4 +1,110 @@
-pandas==1.4.3
-numpy==1.23.1
+anyio==3.7.1
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+arrow==1.2.3
+asttokens==2.2.1
+async-lru==2.0.4
+attrs==23.1.0
+Babel==2.12.1
+backcall==0.2.0
+beautifulsoup4==4.12.2
+bleach==6.0.0
+certifi==2023.7.22
+cffi==1.15.1
+charset-normalizer==3.2.0
+colorama==0.4.6
+comm==0.1.4
+contourpy==1.1.0
```

Comment:
why you add all this modules to requirements?

Reply:
Add requirements manualy